tests/x11/firefox/firefox_html5.pm

```diff
@@ -22,24 +21,8 @@ sub run {
     my ($self) = @_;
     $self->start_firefox_with_profile;
 
-    $self->firefox_open_url('youtube.com/watch?v=Z4j5rJQMdOU');
-    while (check_screen([qw(firefox-youtube-signin firefox-accept-youtube-cookies)], 15)) {
-        if (match_has_tag('firefox-accept-youtube-cookies')) {
-            # get to the accept button with tab and space
-            wait_still_screen(2);
-            send_key_until_needlematch('firefox-accept-youtube-cookies-agree', 'tab', 7, 1);
-            assert_and_click('firefox-accept-youtube-cookies-agree');
-            wait_still_screen(2);
-            next;
-        }
-        elsif (match_has_tag('firefox-youtube-signin')) {
-            assert_and_click('firefox-youtube-signin');
-            wait_still_screen(2);
-            next;
-        }
-        last;
-    }
-    send_key_until_needlematch('firefox-testvideo', 'spc', 30, 5);
+    $self->firefox_open_url('os-autoinst.github.io/os-autoinst-distri-opensuse/testwebsites/html5_video');
```

Comment:
Is this URL valid ?

Reply:
It should be valid after this PR got merged and the GitHub action ran through.

Reply:
@asdil12 can you add a follow up ticket  to have that running in an opensuse.org resource?

Reply:
the python webserver idea is not so bad - I will give it a try